src/Microsoft.Repl/Microsoft.Repl.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <Description>A framework for creating REPLs in .NET Core.</Description>
```

Comment:
Is this meant to be a package that is distributed to NuGet.org for others to use? If so, there are a few more infrastructure hoops to jump through.

Reply:
We talked in person yesterday. Let's not ship Microsoft.Repl.dll as a package yet. The demand is unclear, and we don't want to be on the hook to support the package.

You can disable producing a nupkg by adding `<IsPackable>false</IsPackable>` to this .csproj file.

Reply:
Done